### ISSUE TITLE
Model Files Not Found

### ISSUE THREAD

**Issue by TKTSWalker** [CorentinJ/Real-Time-Voice-Cloning#758]
I tried to create the project and got rid of basically every problem except a Model Files not found problem! It goes as follows:

Arguments:
    datasets_root:    None
    enc_models_dir:   encoder\saved_models
    syn_models_dir:   synthesizer\saved_models
    voc_models_dir:   vocoder\saved_models
    cpu:              False
    seed:             None
    no_mp3_support:   False

********************************************************************************
Error: Model files not found. Follow these instructions to get and install the models:
https://github.com/CorentinJ/Real-Time-Voice-Cloning/wiki/Pretrained-models
********************************************************************************

I'm not sure what i;m doing wrong, I have all three pretrained folders within but i'm still getting the error.

**Comment by AyseEe501** [comment]
Trying to run it on the processor, right?

**Comment by AyseEe501** [comment]
Are you trying to run it on the CPU?

**Comment by AyseEe501** [comment]
I encountered the same problem when I tried to run it on CPU.
And I solved this problem by uninstalling the zip file and trained data and reinstalling.First, I moved the downloaded zip file to C: In order to switch between directories easily. Then I copied the 3 files in the pretrained model I downloaded and pasted it into the file where I will perform the whole function.Then I pasted ffmeg.exe into the same folder. Then I added matplotlib> = 3.2.2 and webrtcvad-wheels in requirements.txt and saved it.All that remains is to proceed through anaconda prompt. So I proceeded from there.conda create -n sound python == 3.7. Then activate it and install , 
=>conda install pytorch torchvision torchaudio cpuonly -c pytorch (pytorch run for conda da cpu),
I went to the directory where I will run demo_toolbox.py. I ran pip install -r requirements.txt and conda install -c conda-forge tensorflow = 1.14 respectively. After that I ran demo_toolbox.py without errors. I hope I can understand and help you
